Policy: fqn-reindex fails with panic and unknown schema

```bash
jake.vanvorhis@jakevanvorhis platform % go run ./service policy fqn-reindex
2024/05/15 11:17:43 INFO trying to resolve absolute path path=/Users/jake.vanvorhis/.opentdf
2024/05/15 11:17:43 INFO adding path to search paths path=/Users/jake.vanvorhis/.opentdf
2024/05/15 11:17:43 INFO trying to resolve absolute path path=.opentdf
2024/05/15 11:17:43 INFO adding path to search paths path=/Users/jake.vanvorhis/desktop/platform/.opentdf
2024/05/15 11:17:43 INFO trying to resolve absolute path path=.
2024/05/15 11:17:43 INFO adding path to search paths path=/Users/jake.vanvorhis/desktop/platform
2024/05/15 11:17:43 INFO attempting to read in config file
2024/05/15 11:17:43 INFO searching for config in paths paths="[/Users/jake.vanvorhis/.opentdf /Users/jake.vanvorhis/desktop/platform/.opentdf /Users/jake.vanvorhis/desktop/platform]"
2024/05/15 11:17:43 INFO creating database client
2024/05/15 11:17:43 ERROR Encountered database error error="ERROR: relation \"opentdf.attribute_namespaces\" does not exist (SQLSTATE 42P01)"
2024/05/15 11:17:43 ERROR Unknown error code error="relation \"opentdf.attribute_namespaces\" does not exist" code=42P01
2024/05/15 11:17:43 ERROR error listing namespaces sql="SELECT opentdf.attribute_namespaces.id, opentdf.attribute_namespaces.name, opentdf.attribute_namespaces.active, JSON_STRIP_NULLS(JSON_BUILD_OBJECT('labels', metadata->'labels', 'created_at', created_at, 'updated_at', updated_at)) AS metadata, opentdf.attribute_fqns.fqn FROM opentdf.attribute_namespaces LEFT JOIN opentdf.attribute_fqns ON opentdf.attribute_fqns.namespace_id = opentdf.attribute_namespaces.id AND opentdf.attribute_fqns.attribute_id IS NULL" error="ERROR: relation \"opentdf.attribute_namespaces\" does not exist (SQLSTATE 42P01)"
panic: could not get namespaces: ERROR: relation "opentdf.attribute_namespaces" does not exist (SQLSTATE 42P01)

goroutine 1 [running]:
github.com/opentdf/platform/service/policy/db.(*PolicyDBClient).AttrFqnReindex(0x1400035fb90)
	/Users/jake.vanvorhis/desktop/platform/service/policy/db/attribute_fqn.go:113 +0x5c0
github.com/opentdf/platform/service/cmd.init.func5(0x104568260, {0x103291ce2?, 0x4?, 0x103291c82?})
	/Users/jake.vanvorhis/desktop/platform/service/cmd/policy.go:45 +0x90
github.com/spf13/cobra.(*Command).execute(0x104568260, {0x104721d80, 0x0, 0x0})
	/Users/jake.vanvorhis/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0x828
github.com/spf13/cobra.(*Command).ExecuteC(0x1045676e0)
	/Users/jake.vanvorhis/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/jake.vanvorhis/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/opentdf/platform/service/cmd.Execute()
	/Users/jake.vanvorhis/desktop/platform/service/cmd/root.go:35 +0x28
main.main()
	/Users/jake.vanvorhis/desktop/platform/service/main.go:6 +0x1c
exit status 2
```

--- a/go.work.sum
+++ b/go.work.sum
@@ -1193,7 +1193,6 @@ github.com/pascaldekloe/goe v0.0.0-20180627143212-57f6aae5913c/go.mod h1:lzWF7FI
 github.com/pelletier/go-toml v1.2.0/go.mod h1:5z9KED0ma1S8pY6P1sdut58dfprrGBbd/94hg7ilaic=
 github.com/pelletier/go-toml v1.8.1/go.mod h1:T2/BmBdy8dvIRq1a/8aqjN41wvWlN4lrapLU/GW4pbc=
 github.com/pelletier/go-toml v1.9.3/go.mod h1:u1nR/EPcESfeI/szUZKdtJ0xRNbUoANCkoOuaOx1Y+c=
-github.com/opentdf/platform/lib/fixtures v0.2.3/go.mod h1:hll1gp/BxPtFqH56YdfvHCUOz4Jk7lhWQSKtU8qptPo=
 github.com/pelletier/go-toml v1.9.5 h1:4yBQzkHv+7BHq2PQUZF3Mx0IYxG7LsP222s7Agd3ve8=
 github.com/pelletier/go-toml v1.9.5/go.mod h1:u1nR/EPcESfeI/szUZKdtJ0xRNbUoANCkoOuaOx1Y+c=
 github.com/pelletier/go-toml/v2 v2.1.0/go.mod h1:tJU2Z3ZkXwnxa4DPO899bsyIoywizdUvyaeZurnPPDc=
@@ -1540,6 +1539,7 @@ golang.org/x/crypto v0.14.0/go.mod h1:MVFd36DqK4CsrnJYDkBA3VC4m2GkXAM0PvzMCn4JQf
 golang.org/x/crypto v0.15.0/go.mod h1:4ChreQoLWfG3xLDer1WdlH5NdlQ3+mwnQq1YTKY+72g=
 golang.org/x/crypto v0.16.0/go.mod h1:gCAAfMLgwOJRpTjQ2zCCt2OcSfYMTeZVSRtQlPC7Nq4=
 golang.org/x/crypto v0.18.0/go.mod h1:R0j02AL6hcrfOiy9T4ZYp/rcWeMxM3L6QYxlOuEG1mg=
+golang.org/x/crypto v0.22.0/go.mod h1:vr6Su+7cTlO45qkww3VDJlzDn0ctJvRgYbC2NvXHt+M=
 golang.org/x/exp v0.0.0-20190121172915-509febef88a4/go.mod h1:CJ0aWSM057203Lf6IL+f9T1iT9GByDxfZKAQTCR3kQA=
 golang.org/x/exp v0.0.0-20190306152737-a1d7652674e8/go.mod h1:CJ0aWSM057203Lf6IL+f9T1iT9GByDxfZKAQTCR3kQA=
 golang.org/x/exp v0.0.0-20190510132918-efd6b22b2522/go.mod h1:ZjyILWgesfNpC6sMxTJOJm9Kp84zZh5NQWvqDGG3Qr8=
@@ -1783,6 +1783,7 @@ golang.org/x/sys v0.12.0/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
 golang.org/x/sys v0.13.0/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
 golang.org/x/sys v0.14.0/go.mod h1:/VUhepiaJMQUp4+oa/7Zr1D23ma6VTLIYjOOTFZPUcA=
 golang.org/x/sys v0.16.0/go.mod h1:/VUhepiaJMQUp4+oa/7Zr1D23ma6VTLIYjOOTFZPUcA=
+golang.org/x/sys v0.19.0/go.mod h1:/VUhepiaJMQUp4+oa/7Zr1D23ma6VTLIYjOOTFZPUcA=
 golang.org/x/telemetry v0.0.0-20240208230135-b75ee8823808 h1:+Kc94D8UVEVxJnLXp/+FMfqQARZtWHfVrcRtcG8aT3g=
 golang.org/x/telemetry v0.0.0-20240208230135-b75ee8823808/go.mod h1:KG1lNk5ZFNssSZLrpVb4sMXKMpGwGXOxSG3rnu2gZQQ=
 golang.org/x/term v0.0.0-20201117132131-f5c789dd3221/go.mod h1:Nr5EML6q2oocZ2LXRh80K7BxOlk5/8JxuGnuhpl+muw=

--- a/service/cmd/policy.go
+++ b/service/cmd/policy.go
@@ -4,6 +4,7 @@ import (
 	"context"
 	"fmt"
 	"log/slog"
+	"strings"
 
 	"github.com/opentdf/platform/service/internal/config"
 	"github.com/opentdf/platform/service/pkg/db"
@@ -63,6 +64,9 @@ var (
 
 func policyDBClient(conf *config.Config) (policydb.PolicyDBClient, error) {
 	slog.Info("creating database client")
+	if !strings.HasSuffix(conf.DB.Schema, "_policy") {
+		conf.DB.Schema += "_policy"
+	}
 	dbClient, err := db.New(context.Background(), conf.DB, db.WithMigrations(policy.Migrations))
 	if err != nil {
 		//nolint:wrapcheck // we want to return the error as is. the start command will wrap it

